### PR TITLE
Add manual deploy docs to README

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Adds to the README details about manual Heroku deployments.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] targeted `dev` branch
- [ ] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] submit a PR against `master` branch after this one (and other PRs depending on this one, e.g. on regen-network/regen-web, if applicable) get merged

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,5 +165,58 @@ the staging or production environment. See `iri-gen/.env-example`.
...
+```
+
+Deploy the branch you want, to the app/remote, i.e. we deploy branch `dev` to
+the app/remote `regen-registry-server-staging`:
+```
+git push regen-registry-server-staging dev:main
+```
+
+Note: When you are deploying a branch, Heroku requires it to be `<branch>:main`
+for some reason.
```
Comment (blushi) [inline README.md:215]: this is because `main` is the name of the remote main branch on heroku for a given app as far as I understand

**Review by blushi — COMMENTED** [review]
Should we also add similar instructions for doing this for master/regen-registry-server?

File: README.md [hunk 2]
```diff
@@ -165,5 +165,58 @@ the staging or production environment. See `iri-gen/.env-example`.
...
+regen-registry-server
+regen-registry-server-staging
+regen-web-backend
+```
+
+Checkout and synchronize the code you want to deploy:
+```
+git checkout dev
+git fetch origin
+git merge origin/dev
```
[the remark this answers is not in the document]
Comment (wgwz) [reply README.md:194]: true, this is just my force of habit/preference.. i can switch it to pull though, it achieves the same purpose here

File: README.md (hunk 1 above)
Comment (wgwz) [reply README.md:215]: it just seemed weird to me because there were no other obviously documented use-cases

**Comment by wgwz** [comment]
> Should we also add similar instructions for doing this for master/regen-registry-server?

sure! will add this.

**Review by blushi — APPROVED** [review]
Thanks for documenting this!

**Review by haveanicedavid — APPROVED** [review]
Looks great!

**Pull request merged**